src/MainTest.java

```diff
@@ -0,0 +1,96 @@
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.junit.Before;
+import org.junit.Test;
+import static org.junit.Assert.assertEquals;
+
+
+public class MainTest {
+    private JSONArray jsonArray;
+
+    @Test
+    public void testFormatDate_ValidTimestamp() {
+        // Test the formatDate method with a valid timestamp value.
+        long timestamp = 1079931600;
+        String expected = "22/03/2004 05:00:00"; // GMT Mon Mar 22 2004 05:00:00 GMT+0000
+        String actual = Main.formatDate(timestamp);
+
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    public void testFormatDate_ZeroTimestamp() {
+        // Test the formatDate method with a zero timestamp value.
+        long timestamp = 0;
+        String expected = "01/01/1970 00:00:00"; // GMT Thu Jan 01 1970 00:00:00 GMT+0000
+        String actual = Main.formatDate(timestamp);
+
+        assertEquals(expected, actual);
+    }
+
+
+    @Before
+    public void setUp() {
```

Comment:
setup should be at the beggining of the class. also you may shrink the inputs, I think it's too much for tests, just 3-4 records